airflow/providers/google/cloud/utils/field_validator.py

```diff
@@ -421,6 +421,8 @@ def validate(self, body_to_validate: dict) -> None:
         :param body_to_validate: body that must follow the specification
         :return: None
         """
+        if body_to_validate is None:
```

Comment:
The function’s type annotation should probably be updated to cover the None case.

Reply:
I am slightly confused if it would make sense to update the type annotation to include None as an acceptable type only to raise an error. 🤔